packages/components/src/components/scroller/scroller-styles.ts

```diff
@@ -81,31 +89,23 @@ export const getComponentCss = (
         border: 0,
         outline: 0,
         cursor: 'pointer',
-        background: gradientColorTheme === 'surface' ? backgroundSurfaceColor : backgroundColor,
+        background: backgroundColorMap[theme][gradientColorTheme],
         borderRadius: borderRadiusSmall,
         ...frostedGlassStyle,
         visibility: 'hidden',
         ...(!isDarkTheme && dropShadowLowStyle),
         ...hoverMediaQuery({
           transition: getTransition('background-color'),
           '&:hover': {
+            ...frostedGlassStyle,
             background: hoverColor,
-            ...(isDarkTheme && {
-              '& > .icon': {
-                filter: 'invert(97%) sepia(55%) saturate(2840%) hue-rotate(180deg) brightness(114%) contrast(103%)', // TODO: this is not shared from icon?
-              },
-            }),
           },
         }),
       },
     },
     root: {
       display: 'grid',
-      gridTemplateColumns: '48px minmax(0, 1fr) 48px',
-      ...hoverMediaQuery({
-        // distinguish gradient width on mobile and desktop
-        gridTemplateColumns: '64px minmax(0, 1fr) 64px',
-      }),
+      gridTemplateColumns: '3rem minmax(0, 1fr) 3rem',
```

Comment:
Is `rem` a good choice?

Reply:
If we dont use rem here and we scale the component, the button gets bigger but the gradient not! The Gradient and the button are in the same box and should both be scaled in my opinion. You have a different scroll position if you scale the gradient, for sure, but if you dont the prev/next button would cover parts of the active step.